### ISSUE TITLE
IPFS

### ISSUE THREAD

**Issue by sanchez** [draganmarjanovic/DBNB#6]
Good shit

**Comment by draganmarjanovic** [comment]
@sanchez I'm having trouble with some of the JS / was hoping you could take a look.

I can successfully publish to IPFS but am having trouble getting the hash back and passing it on to be stored in the blockchain. These lines show the code I try to use to do this.

https://github.com/draganmarjanovic/DBNB/blob/09b1c1230fe7eae40f2c983061826db126036473/src/pages/Account.js#L35-L42

I also can't work out why I'm getting errors when I try to use `this.getAccountID()` as is used in all the other `send()` methods. See:
https://github.com/draganmarjanovic/DBNB/blob/09b1c1230fe7eae40f2c983061826db126036473/src/lib/AccountManagement.js#L243-L244

Basically what I see as the only part missing from this fully functioning is passing that hash that is returned from `IpfsUtils.publish()` onto the blockchain as imageLocation in the Account contract. 😭😫

**Comment by sanchez** [comment]
From the page it isn't `this.getAccountID()`, it is the account object you need. So `this.props.account.getAccountID()`

**Comment by draganmarjanovic** [comment]
@sanchez it's not from the page though, if you go into that and scroll slightly up, the other methods are using `this.getAccountID()`. Maybe I'm missing something but it seems weird...?
